src/actions/articleActions/rateActions/index.js

```diff
@@ -0,0 +1,46 @@
+import { toast } from "react-toastify";
+import { ArticleActionTypes } from "../../types";
+
+const userToken = localStorage.getItem("token");
+const { BASE_URL } = process.env;
+
+const RateAction = (rate, article) => dispatch => fetch(`${BASE_URL}/articles/${article.slug}/rate/`, {
```

Comment:
Hey @BagzieGracious, for consistency, Let's consider keeping function names descriptive and camel-cased.